policy/compressor.go

```diff
@@ -0,0 +1,132 @@
+package policy
+
+import (
+	"sync"
+
+	"github.com/m3db/m3cluster/kv"
+	"github.com/m3db/m3metrics/generated/proto/schema"
+	"github.com/m3db/m3x/instrument"
+	"github.com/m3db/m3x/log"
+)
+
+// Compressor maintains a mapping of policies to integers. It is useful when encoding policies
+// so that an integer representation of a policy can be sent instead of the entire Policy
+type Compressor interface {
```

Comment:
Yea, since there's not too many interfaces in the policy package I think it's simpler to leave them in their respective files.

I'd also prefer to leave the interfaces separate since it's a little cleaner imo. I don't think the Aggregator will need to encode metrics, just decode them, for the time being.